src/components/forms/SelectOptionInterface.ts

```diff
@@ -0,0 +1,21 @@
+import {SelectEntry} from "src/components/forms/SelectOptions";
+
+export interface SelectOptionWithKeyNameAndAbbreviation {
+    key: string,
+    name?: string,
+    abbreviation?: string
+}
+
+export function getFormattedName(entry: SelectOptionWithKeyNameAndAbbreviation) {
+    if (entry.abbreviation && entry.name) {
+        return `${entry.name} (${entry.abbreviation})`;
+    }
+    if (entry.name) {
+        return `${entry.name}`;
```

Comment:
How is this different than `entry.name`? `name` is already `string?` and its type will be `string` since it's inside `if (entry.name) {`.
```suggestion
        return entry.name;
```

Reply:
It's a code style habit from another project.  Adjusted.